### ISSUE TITLE
Translation Gang integration

### ISSUE THREAD

**Issue by gbezyuk** [vuejs-fr/vuejs.org#19]
Salut! Unfortunately my French is almost non-existent, so I'll use English if you don't mind.

On behalf of @translation-gang I want to invite you to be a French part of the community.

Translating to different languages have many same technical issues, so we could benefit from maintaining translations under the same team. For instance, it's easier to detect changes once in order to keep all the translations up-to-date.

Merci!

**Comment by sylvainpolletvillard** [comment]
Hi @gbezyuk 

Having a common space for all Vue.js translators seems a good idea. How is your community currently working and what do you expect from us?

**Comment by gbezyuk** [comment]
Hi @sylvainpolletvillard! Currently we maintain only Russian translations, but we are looking forward to going international, which is why I'm here. Basically, we have forked repos for every translated project (which for now are vuejs.org, vue-router, vuex, nuxt and vee-valitate), a core team of translators/developers and many outside contributors. First of all, we have translated and proof-read the original docs, and then we follow the changes and update the translations accordingly, so far so good. Actually, a couple of days ago we've even got ahead of the original docs, having a PR with translation of new stuff at the same time as the original repo had the same PR in English.

The main ideas are:
- to have a common space for all the volunteer translators
- to reduce the risks of leaving translations unattended
- basically, to establish a culture of open source translations and endorse the idea of having good quality docs synced with the original ones for every important language and every important project
- to hep junior developers via providing docs in their native language

**Comment by sylvainpolletvillard** [comment]
Okay, let us know what we can do to help.
Here is the current team for French translation: @Haeresis @nyl-auster @Teraglehn @gbaudoin @zyhou

**Comment by gbezyuk** [comment]
Well, I assume we could invite the French team to TG core, moving your repos there too, as fr.vuejs.org. Then you just continue the translation, and when it's done all further original doc updates will be triggering creating issues for all translation repos. For instance, if some files were changed in the original docs and Russian team has noticed it, they'll create an issue for translation not only in the Ru repo, but in the Fr one as well — so we don't need to watch the changes twice.

Are you guys okay with this plan?

**Comment by Haeresis** [comment]
Hello @gbezyuk,

I'm @haeresis and I'm one of Vuejs-FR Organization.

I have understand the purpose of Translation Gang I think, so it's a place of all translations team can traduce some projects in theirs languages and are up-to-date one others, and Vue is a good starting point.

My voice not reflect the entire voice of Vuejs-FR because here, there are no « main maintainer », and there are not a only one person that decide to do something but I will expose my vision.

Currently, I endorsed the role of person that check modification from vue/vuejs.org and kept translation up to date at https//fr.vuejs.org/ and I will explain why Vuejs-FR was created. But I'm not necessarily the only one and not it's not necessarily me will can do that forever, it's Vuejs-FR.

So because Vuejs-FR have currently a « translation team », it's also possible it have some others teams for marketing, philosophie etc that can not have place under Translation Gang.

## Network vs Hierarchy  ##

What I « love / like » with Github, it is not a pyramidal hierarchy with one person/group than drive subgroup, than drive subgroup, than drive subgroup,... It's an ecosystem of self created group with « motors / desires / wishes ». And it's natural that, each culture, prefer or not this vision, is interested to traduce things from english to its culture or not, is more interested by network communauty or hierarchical communauty.

I'm not think it's a problem if french team are not up-to-date with the russian team and not up-to-date with the japan team etc. The most important thing it's « when a team from X culture send some « input/demand/interaction » to the Y culture, for advice, sharing things, sharing ideas, the Y culture respond with output in the same way but it's not a necessity to have « one brain » that drive « all brains ». It's an illusion, network management can embrace severals differents cultures and idea that are too big enouth for the only one guy at the top of communauty that decide. **I prefer some technique like "advice process"**.

For exemple here : https://github.com/vuejs/vuejs.org/blob/master/README.md we not said « Primary maintainer - X Person » because we are a network entity. And in this document there is no mention of « one guy/group » that embrace all traduction : and I understand it is your « motor/desire/wish » with Translator Gang and I respect that, really I understand your point of view.
 
## Vuejs Team vs Translation Team ##

I'm not here because I'am a person who  « love / like » translation and « inter-translation communauty » but I'm here because I'am a person who « love / like » Vuejs and « love / like » french culture. So I'm not personnaly interested to say « Hey, I'm one of French translator of Translator Gang » but I'm interested to say « Hey, I'am one of Vuejs "lovers / likers". It's because here I want help an care about person that « Love/Like Vuejs + Love/Like French Culture » and not to be a part of Translation Gang.

If the world have so many culture, and if I'm here to do some effort to « translate » Vuejs it's because the French culture is really different from Russian culture and from English culture so our purpose is not kept culture code of original language for Vuejs, it's to kept concept and adapt it to our culture.

## Someone new to Vuejs and French ##

If someone to new here or with Vuejs want share with his culture, it's more interesting for it the issue are under Vuejs-FR, the forum are under Vuejs-FR, the giter are under vuejs-fr/vue and not under Translation Gang.

## Concept of Culture ###

And more personnally I think it's an « anti-communautary » idea, it's not the serendipity concept that it's the concept of Github, NPM, and them of « The FR Team of » or « The RU Team of ». I don't have the word, it's more a feeling but for example: 

- It's not the goal of one entity to decide if for the communauty X it's a better thing to use gitter or another plateform for chat support for this communauty, but it's the communauty it-self because it « bear/have/is » the culture.

- It's the Russian communauty that is the better person to choose if it'is more interesting to share something one Facebook or on VK (if a not do mistake on the purpose of VK and the original language of service).

So I definitively prefer, and I think it's preferable each team have each communauty because it's the heart of github to share effort and share guideline from the better working group and Under the communauty for it (here it's Vuejs-FR as [ask recommanded by Evan You](https://github.com/vuejs/vuejs.org/issues/271#issuecomment-266476746)).

*Share guideline, idea, provide some recommandation or maintain a project and the purpose of this project for « coordinate all translation » or « coordinate all traduce » between culture have senses but have all cultures under one group not for the reason that a culture is an autonome unit. It's precisely what culture and translation group exist in each culture.*

**So I vote**
- Yes for sync all efforts in Translation Team issue in the mesure of disponibility of each person and with Translation Team guide if Original Vuejs Culture want all traduction coordination are do by this team.
- but **No to move Vuejs-Fr communauty under a « mondial culture » project.**

But, it's a collective decision so it's not definitive.

Good luck for this project that motive you @gbezyuk !

**Comment by sylvainpolletvillard** [comment]
Not sure I get the whole picture here but I also don't understand the need for moving our repository. If somehow the new doc changes are automatically detected and an issue is created on the TG repo, we can always catch up the changes there. But keeping up with doc changes has never been a big deal since it is easy to get diffs on specific date ranges and folders with git.

@Haeresis you use the word "culture" about a thousand times :smile: I don't think TG group aims for a "monoculture", as far as I understood this is mostly about organisation and communication. I don't want to turn this into a go/no go situation or even worse a cultural debate. Let's just focus onfriends on Github and find ways to help each other. Our little team is working pretty well at the moment so there is no need to reinvent the whole thing, neither totally shutting off the door to future collaborations.

**Comment by gbezyuk** [comment]
Okay guys ) let's keep it as it is.
